pdfarranger/pdfarranger.py

```diff
@@ -1573,7 +1574,7 @@ def paste_as_layer(self, data, destination, laypos, offset_xy=None):
                     lp.offset[i] += outside
```

Comment:
The lines lp.crop[i] +=  and lp.offset[i] +=  cause "TypeError: 'Sides' object does not support item assignment"
To test this code you need to copy and paste as overlay a page which already has a overlay.

Reply:
Thank you, I'll add a test for that

Reply:
Sorry about this. #960 should fix it